jobs/JEVS_AQM_PLOTS

```diff
@@ -59,13 +61,11 @@ export VDATE=${VDATE:-${PDYm3}}
 
 export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver}/)}
 export COMINaqm=${COMINaqm:-$(compath.py ${envir}/com/${NET}/${evs_ver}/stats/${COMPONENT}/${MODELNAME})}
-export COMOUT=${COMOUT:-${COMIN}/${STEP}/${COMPONENT}}
-export COMOUTplots=${COMOUTplots:-${COMOUT}/${RUN}.${VDATE}}
-export COMOUTheadline=${COMOUTheadline:-${COMOUT}/headline.${VDATE}}
+export COMOUT=${COMOUT:-$(compath.py -o ${envir}/com/${NET}/${evs_ver}/)}
```

Comment:
When compath.py is used w/ -o, do not use `${envir}/com`. Please update this definition to be: `export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${evs_ver}/)}`

Reply:
Fixed.